File: javaScript/problem-solving/prime-number.js
function primeNumber(num) {
  switch (num) {
    case 2:
    case 3:
    case 5:
    case 7:
      console.log(`${num} is a prime number`);
      break;
    default:
      console.log(`${num} is not a prime number`);
  }
}

primeNumber(2);
primeNumber(3);
primeNumber(5);
primeNumber(7);
primeNumber(8);

File: python-problem-solving/prime-number.py
# Find prime number using python


def prime_number(num):
    match num:
        case 2 | 3 | 5 | 7:
            return print(f"{num} is a prime number")
        case _:
            return print(f"{num} is not a prime number")


prime_number(2)
prime_number(3)
prime_number(5)
prime_number(7)
prime_number(8)
